### PR TITLE
IndyAV: Added initial directories for the software.

### PR DESCRIPTION
**Pull request by popeyedpanther**
Testing out the process and creating some initial directories.

### CODE REVIEW

**Comment by MarshallRawson** [comment]
Please squash your commits into one commit

**Comment by kev-the-dev** [comment]
@MarshallRawson you don't mind that there is a duplicated file in this diff?

**Comment by MarshallRawson** [comment]
> @MarshallRawson you don't mind that there is a duplicated file in this diff?

oh ya are you talking about `docs/indyav/index.notrst`

**Comment by popeyedpanther** [comment]
I left the file to keep the folder tracked. I will modify it when the docs becomes more used.

File: IndyAV/index.rst [hunk 1]
```diff
@@ -0,0 +1,12 @@
+Indy AV
```
Comment (MarshallRawson) [inline IndyAV/index.rst]: Why not use this as an opportunity to talk about indyAV project and provide some links (like to the competition website / Dr. Crane's website)

File: IndyAV/localization/README.md [hunk 2]
```diff
@@ -0,0 +1,3 @@
+# Localization
```
Comment (MarshallRawson) [inline IndyAV/localization/README.md]: Talk about the sylphase and 3d lidar mapping that we already have demonstrated and strengths/weaknesses / strategy for how we are going to localize

File: IndyAV/perception/README.md [hunk 3]
```diff
@@ -0,0 +1,3 @@
+# Perception
```
Comment (MarshallRawson) [inline IndyAV/perception/README.md]: Why not talk about the preception requirments that we know we will have to do for this project and the various way to get there

File: IndyAV/planning/README.md [hunk 4]
```diff
@@ -0,0 +1,2 @@
+# Planning
```
Comment (MarshallRawson) [inline IndyAV/planning/README.md]: Why not talk about what planning we are going to need

File: IndyAV/simulation/README.md [hunk 5]
```diff
@@ -0,0 +1,2 @@
+# Gazebo Simulation
```
Comment (MarshallRawson) [inline IndyAV/simulation/README.md]: Why not talk about gazebo and how other people do the same thing with gazebo

**Comment by popeyedpanther** [comment]
I don't know why this failed to build. It has nothing to do with the changes made.

**Pull request closed without merging**